src/test/java/uk/gov/companieshouse/api/testdata/spec/CompanySpecTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.companieshouse.api.testdata.spec;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.testdata.model.rest.CompanySpec;
+import uk.gov.companieshouse.api.testdata.model.rest.Jurisdiction;
+
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+class CompanySpecTest {
+
+    private final ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+    private Validator validator;
```

Comment:
Maybe we don't need to declare this as an instance field if it's only used locally within some test methods?

Reply:
moved this to method level